client/watcher/watcher.go

```diff
@@ -74,8 +74,11 @@ func (w *Watcher) Watch() {
 			w.mu.Lock()
 			select {
 			case event := <-w.watcher.Events:
-				if event.Op&fsnotify.Write == fsnotify.Write {
-					w.UpdateBytes()
+				if event.Op&fsnotify.Write == fsnotify.Write || event.Op&fsnotify.Create == fsnotify.Create {
+					err := w.UpdateBytes()
+					if err != nil {
+						printer.LogErrf("[dcdr] watch error: %v", err)
```

Comment:
nit: can we call this an `updateBytes` error or something to differentiate from L84?